auth/src/test/java/com/firebase/ui/auth/ui/email/WelcomeBackPasswordPromptTest.java

```diff
@@ -39,76 +40,47 @@
 import org.junit.runner.RunWith;
 import org.robolectric.Robolectric;
 import org.robolectric.RuntimeEnvironment;
-import org.robolectric.Shadows;
 import org.robolectric.annotation.Config;
-import org.robolectric.shadows.ShadowActivity;
 
 import java.util.Collections;
 
 import static com.firebase.ui.auth.testhelpers.TestHelper.verifySmartLockSave;
-import static junit.framework.Assert.assertEquals;
-import static org.junit.Assert.assertNull;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.reset;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
 @RunWith(CustomRobolectricGradleTestRunner.class)
 @Config(constants = BuildConfig.class, sdk = 23)
-public class SignInActivityTest {
+public class WelcomeBackPasswordPromptTest {
     @Before
     public void setUp() {
         TestHelper.initializeApp(RuntimeEnvironment.application);
         PlayServicesHelper.sApiAvailability = TestHelper.makeMockGoogleApiAvailability();
     }
 
-    private SignInActivity createActivity() {
-        Intent startIntent = SignInActivity.createIntent(
+    private WelcomeBackPasswordPrompt createActivity() {
+        Intent startIntent = WelcomeBackPasswordPrompt.createIntent(
                 RuntimeEnvironment.application,
                 TestHelper.getFlowParameters(Collections.<String>emptyList()),
-                null);
+                new IdpResponse(EmailAuthProvider.PROVIDER_ID, TestConstants.EMAIL));
         return Robolectric
-                .buildActivity(SignInActivity.class)
+                .buildActivity(WelcomeBackPasswordPrompt.class)
                 .withIntent(startIntent)
                 .create()
                 .visible()
                 .get();
     }
 
     @Test
-    public void testSignInButton_validatesFields() {
```

Comment:
Shouldn't `WelcomeBackPasswordPrompt` validate fields now that we're removing `SignInActivity`?